__tests__/app.test.js

```diff
@@ -56,9 +56,9 @@ describe("/api", () => {
             );
           });
       });
-      test("Should accept a query for category to sort the reviews by", () => {
+      test("Should accept a query for category to filter the reviews by", () => {
         return request(app)
-          .get("/api/reviews?sort=dexterity")
+          .get("/api/reviews?filter=dexterity")
```

Comment:
I think something like category=dexterity would be more appropriate here. A sort_by can refer to many different things, but  our filter will only ever refer to the topic. If there were an alternative filter option (for example, "author"), we could do something like author=authorname.